### DISCUSSION TITLE
Can no longer compile simple A2DP sink with i2s pin config

### DISCUSSION THREAD

**Discussion by Aalexz** [pschatzmann/ESP32-A2DP#586] [Q&A] [answered]
Can no longer compile simple A2DP sink with i2s pin config, the error I get is 

Compilation error: 'class BluetoothA2DPSink' has no member named 'set_pin_config'

A2DP_LEGACY_I2S_SUPPORT is set to true and no change.

**Comment by pschatzmann** [comment, marked as answer]
Just read the Readmy and the [Wiki](https://github.com/pschatzmann/ESP32-A2DP/wiki/Legacy-I2S-API)!

set_pin_config is only available if you use an Arduino ESP core version where the Legacy API is available!

**Comment by Aalexz** [comment]
Sorry Phil and thanks for the great library. It used to work and now it
doesn't - something which is frustrating when this forms part of a larger
project. I looked in the header file at least ey.

Best regards Alex

**Comment by pschatzmann** [comment]
I assume you made an upgrade of the ESP32 core and I can't provide you the functionality that Espressif has removed.
If you want it to work the old way you need to go back to an old Arduino ESP32 core release.

But if you want to future-proof your project, you need to adjust to the new functionality.

**Comment by Aalexz** [comment]
Thanks for the info Phil, I did not know it was Espressif that had made the
change. Again, I know it really isn't worth your time to respond to folks
like me who admittedly didn't even read the readme, and thank you for your
support.

Alex
